test/spec/app_spec.rb

```diff
@@ -1031,21 +1026,44 @@
       expect(title.text == 'Not Found').to be(true)
     end
 
-    # it "should cancel request through switch teams" do
-    #   skip("Needs to be implemented")
-    # end
-
-    # it "should linkify URLs on comments" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should cancel request through switch teams", bin1: true do
+      user = api_register_and_login_with_email
+      t1 = api_create_team(user: user)
+      t2 = api_create_team(user: user)
+      page = MePage.new(config: @config, driver: @driver).load
+          .select_team(name: t1.name)
+      wait_for_selector("team-members__edit-button",:class)        
+      expect(page.team_name).to eq(t1.name)
+      page = MePage.new(config: @config, driver: @driver).load
+          .select_team(name: t2.name)
+      wait_for_selector("team-members__edit-button",:class)        
+      expect(page.team_name).to eq(t2.name)
+    end
 
-    # it "should add and remove suggested tags" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should linkify URLs on comments", bin1: true do
+      media_pg = api_create_team_project_and_claim_and_redirect_to_media_page
+      expect(@driver.page_source.include?('Your comment was added!')).to be(false)
+      old = wait_for_selector_list("annotation__default-content",:class).length     
+      fill_field('textarea[name="cmd"]', 'Go to https://meedan.com/en/')
+      el = wait_for_selector("//span[contains(text(), 'Submit')]", :xpath)
+      el.click
+      old = wait_for_size_change(old, "annotation__default-content", :class)
+      expect(@driver.page_source.include?('Your comment was added!')).to be(true)
+      el = wait_for_selector_list("//a[contains(text(), 'https://meedan.com/en/')]", :xpath)
+      expect(el.length == 1).to be(true)
+    end
 
-    # it "should find all medias with an empty search" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should find all medias with an empty search", bin1: true do
+      api_create_media_and_go_to_search_page
+      old = wait_for_selector_list("medias__item", :class).length
+      el = wait_for_selector("search-input", :id)
+      el.click
+      @driver.action.send_keys(:enter).perform
+      sleep 3 #due the reload
+      wait_for_selector("search-input", :id)
+      current = wait_for_selector_list("medias__item", :class).length
+      expect(old == current).to be(true)
```

Comment:
Would be good to check that, besides being equal, they are greater than zero.

Reply:
Implementing